invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DangerZone.js

```diff
@@ -15,49 +15,68 @@ import { RenameCommunitySlugButton } from "./RenameCommunitySlugButton";
 import PropTypes from "prop-types";
 import { DeleteCommunityModal } from "./DeleteCommunityModal";
 
-const DangerZone = ({ community, onError }) => (
-  <Segment className="negative rel-mt-2">
-    <Header as="h2" className="negative">
-      {i18next.t("Danger zone")}
-    </Header>
-    <Grid>
-      <Grid.Column mobile={16} tablet={10} computer={12}>
-        <Header as="h3" size="small">
-          {i18next.t("Change identifier")}
+const DangerZone = ({ community, onError, permissions }) => {
+  if (permissions.can_delete || permissions.can_rename) {
```

Comment:
do we need to change the permissions generator on the service? Maybe only in Zenodo? because if you are community owner by default you can still change the name, and I understood we wanted to avoid this

Reply:
We override the permissions only in Zenodo: https://github.com/zenodo/zenodo-rdm/pull/546